documentation/asciidoc/computers/raspberry-pi/boot-http.adoc

```diff
@@ -0,0 +1,50 @@
+== HTTP Boot
+
+The network install feature uses HTTP over ethernet to boot the device into the xref:getting-started.adoc#using-raspberry-pi-imager[Raspberry Pi Imager] application.
+
+In addition to network install you can explicitly boot your device with files downloaded via HTTP with xref:raspberry-pi.adoc#BOOT_ORDER[boot mode] `7`. You can still use this even if xref:raspberry-pi.adoc#NET_INSTALL_ENABLED[network install / keyboard detection on boot is disabled].
+
+You could for example add this to your `BOOT_ORDER` as a fall back boot method or put it behind a GPIO conditional to initiate HTTP download from your own server when a GPIO is pulled low. e.g.
+
+```
+[gpio7=0]
+BOOT_ORDER=0xf7
+HTTP_HOST=downloads.raspberrypi.org
+NET_INSTALL_ENABLED=0
+```
+
+In this case if GPIO 7 were to be pulled low, the files http://downloads.raspberrypi.org:80/net_install/boot.img and http://downloads.raspberrypi.org:80/net_install/boot.boot.sig would be downloaded. If network install / keyboard detection on boot were enabled it would use the same url. If GPIO 7 were not pulled low the behaviour would be unchanged.
+
+HTTP in the `BOOT_ORDER` will be ignored if secure boot is enabled and xref:raspberry-pi.adoc#HTTP_HOST[HTTP_HOST] is not set.
+
+=== Requirements
+
+To use HTTP boot you need to xref:raspberry-pi.adoc#bootloader_update_stable[use the LATEST / STABLE bootloader configuration] and update to a bootloader dated 10th March 2022 or later. HTTP boot only works over ethernet, so you need an Ethernet cable connected to the network, e.g. via a socket on the back of your router.
```

Comment:
"... so you need to connect your Raspberry Pi to your network via an Ethernet cable, e.g. ..."